backend/template_mail.json

```diff
@@ -0,0 +1,14 @@
+{
+    "emails": {
+        "grant_access": {
+            "subject": "Welcome to Virtual Labs IIIT Hyderabad - Workshop Tracker Access",
+            "body": "Dear Nodal Coordinator,\nGreetings from Virtual Labs!\nWe are delighted to welcome your college as a Nodal Centre for Virtual Labs IIIT Hyderabad. Thank you for your interest in this initiative.\nYou have been granted access to our Workshop Tracker. Use the Nodal Coordinator email from your Expression of Interest document as your login credentials.\nThe portal allows you to upload workshop-related data for your Nodal Centre and View other Nodal Centres associated with Virtual Labs IIIT Hyderabad. Please refer the user document for reference.\nFor any queries, please contact us at ioc-iiith@vlabs.ac.in.\nWe look forward to a successful collaboration.",
+            "html": ""
```

Comment:
What is the order of processing of body and html. Should there be only one of them in the config?
It should be clearly documented

Reply:
body will be processed only if html is an empty string. I have added this information to the tech-doc.